### PR TITLE
Update Staff

### PR DESCRIPTION
**Pull request by CannonLock**
Remove Lauren 

Add Mike and Theng

### CODE REVIEW

File: _data/team/Micheal-Collins.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: Micheal Collins
+shortname: clock
```
Comment (ChristinaLK) [inline _data/team/Micheal-Collins.yml:2]:
```suggestion
shortname: mcollins
```

File: _data/team/Micheal-Collins.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+name: Micheal Collins
```
Comment (ChristinaLK) [inline _data/team/Micheal-Collins.yml:1]:
```suggestion
name: Michael Collins
```

**Review by ChristinaLK — COMMENTED** [review]
a few typos - you may also want to rename Michael's yaml file to have the correct spelling. 

could you just change Lauren's profile from active to inactive instead of deleting?

**Comment by CannonLock** [comment]
@ChristinaLK Active doesn't actually do anything so deleting is the only option. Are you looking to have the information preserved somewhere?

**Comment by ChristinaLK** [comment]
@CannonLock not really, just seems like a harsh cutoff. :P

**Comment by CannonLock** [comment]
@ChristinaLK Yeah agreed. At least it is less harsh then if this got merged when I made the PR the day after she left :)

**Review by ChristinaLK — APPROVED** [review]

**Pull request merged**